libkpod/kill.go

```diff
@@ -0,0 +1,32 @@
+package libkpod
+
+import (
+	"github.com/docker/docker/pkg/signal"
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/kubernetes-incubator/cri-o/utils"
+	"github.com/pkg/errors"
+	"os"
+)
+
+// ContainerKill kills ones or more containers with a user provided signal
```

Comment:
Sends the user provided signal to the containers primary process.

Reply:
fixed.